src/shared/hooks/useDataApi.ts

```diff
@@ -15,7 +15,7 @@ export type UseDataAPIState<T> = {
   data?: T;
   status?: AxiosResponse['status'];
   statusText?: AxiosResponse['statusText'];
-  headers?: AxiosResponse['headers'];
+  headers?: Record<string, string>;
```

Comment:
Did you have to change this because of custom headers? Why is it an issue now but wasn't before?

Reply:
`AxiosResponse['headers']` is typed as `any` in the library definition package, which is why it didn't block us before, we could quite literally do `any`thing with this value.
I just made it more restrictive here after seeing that I had made an error and TS hadn't caught it.